roles/rciam-probes/handlers/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Clean cache
+  command: "{{ clean_cache_cmd }}"
+  listen: "yum Update"
+  become: yes
+
+- name: yum Update (RedHat)
+  yum:
+    name: '*'
```

Comment:
Upgrading all OS packages can cause unpredictable effects. Can we avoid this?

Reply:
This is now a task which only fire with the tag upgrade. We need this when deploying for the very first time.